automation/project/Dependencies.scala

```diff
@@ -15,7 +15,7 @@ object Dependencies {
   val workbenchGoogle: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-google" % workbenchGoogleV excludeAll excludeWorkbenchModel
   val excludeWorkbenchGoogle = ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = "workbench-google" + scalaV)
 
-  val workbenchServiceTestV = "0.21-258d483"
+  val workbenchServiceTestV = "0.21-e8c8c3e1-SNAP"
```

Comment:
will change to non-snapshot when https://github.com/broadinstitute/workbench-libs/pull/892 is merged